src/dndbeyond/content-scripts/character.js

```diff
@@ -568,6 +568,23 @@ function handleSpecialGeneralAttacks(damages=[], damage_types=[], properties, se
     }
 
     // Class Specific
+    if (character.hasClass("Fighter")) {
+        if(character.hasClassFeature("Psionic Power")) {
+            // HACK ALERT: fixes dndbeyond missing mods but incase they are added in the future we ensure the mod is applied if not present if it is present it is not applied
+            const intelligence = character.getAbility("INT") || {mod: 0};
+            const mod = parseInt(intelligence.mod) || 0;
+            const psychic_action = action_name.toLocaleLowerCase();
+            if(["psionic power: psionic strike"].includes(psychic_action)) {
+                // Use full modifier, even if deeply negative
+                damages[0] = ensureModifier(damages[0], mod);
+            } else if(["psionic power: protective field"].includes(psychic_action)) {
+                // Clamp the modifier to minimum of -1, not below that
+                const cappedMod = mod < -1 ? -1 : mod;
+                damages[0] = ensureModifier(damages[0], cappedMod);
```

Comment:
This is the wrong fix, you shouldn't be changing the modifier, because if it's -3, and you roll a 20, you should have a total of 17, not 19.
Instead, this was my suggestion:
```js
// We set the minimum dice roll to be the minimum value needed so that the minimum total with the mod is 1
const min = mod >= 0 ? 1 : Math.abs(mod) + 1;
damages[0] = ensureModifier(damages[0], mod).replace(/[0-9]*d[0-9]+/g, `$&min${min}`)`;
```